tests/integration/test_bayesian_optimization.py

```diff
@@ -65,7 +67,7 @@
             (20, EfficientGlobalOptimization()),
             (25, EfficientGlobalOptimization(AugmentedExpectedImprovement().using(OBJECTIVE))),
             (
-                15,
+                25,
```

Comment:
The test broke for me on my machine when it was 15...

Reply:
Looking into this further, I think this is actually because we had to upgrade GPflux to 0.2.*, and probably something changed in the way it does Fourier features from 0.1.*. Any suggestions?

Reply:
there were some changes in efficient sampling, whitening is now an option (check the `sample.py` file: https://github.com/secondmind-labs/GPflux/compare/26b103c2df1ed3868da81d053109332485fb0e5b...v0.2.3), but by default its set to `False`, is whitening used here in the test? 

could it be just your machine? do the tests on GH also now fail here?

Reply:
The same test on the `develop` branch now fails for me as well, so it seems like it's just something with my machine again.